keyboards/40percentclub/ut47/rules.mk

```diff
@@ -34,7 +34,7 @@ FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 CUSTOM_MATRIX = yes
 SRC += matrix.c protocol/serial_uart.c
 
-ifeq ($(strip $(LED_ENABLE)), yes)
+ifeq 
```

Comment:
Looks like your method of automating the removal is a touch too aggressive and is causing various boards to fail compilation.

Reply:
That's embarrassing. Looks like I need to be more specific with the regex.